src/app/components/privacy-policy/privacy-policy.component.scss

```diff
@@ -36,21 +34,21 @@
       }
 
       li.scroll-selected {
-        color: $red-light;
```

Comment:
you can directly change the red-light variable. Also I think bit darker color will be better to read the text.

Reply:
@Shekharrajak done